HyperV/scripts/gerrit-git-prep.sh

```diff
@@ -102,6 +102,7 @@ then
     else
         git clone $GIT_ORIGIN/$ZUUL_PROJECT .
     fi
+    ls -a
```

Comment:
it should also include a pwd or something to identify the exact location.
On line 79, we should not remove the build dir since we're working only with the project dir...